### DISCUSSION TITLE
Not Authorized - Tell me why ?

### DISCUSSION THREAD

**Discussion by a-harbuz** [aws/aws-cli#8691] [Q&A]
PS C:\Users\user\source\repos> aws ecr-public get-login-password --region us-east-1 | docker login --username ****** --password-stdin ***** public.ecr.aws/o8s2m5j2
>>
accepts at most 1 arg(s), received 3
PS C:\Users\user\source\repos> docker tag first:latest public.ecr.aws/o8s2m5j2/first:latest
PS C:\Users\user\source\repos> docker push public.ecr.aws/o8s2m5j2/first:latest
The push refers to repository [public.ecr.aws/o8s2m5j2/first]
fb2f2b8069c8: Preparing
10359c5dc4ba: Preparing
601b48657e0c: Preparing
b42107e74152: Preparing
denied: Not Authorized

Not Authorized - Tell me why ?????

I already made:
Set-AWSCredential `
             -AccessKey ***** `
             -SecretKey ***** `
             -StoreAs MyNewProfile

docker push , Not Authorized - Tell me why ?????

I used the standard instructions from Amazon Elastic Container Registry !!!

**Comment by verystrongvery** [comment]
Have you solved this problem now?

**Comment by a-harbuz** [comment]
my account is blocked. Just unblock it. I can't use ECS either. The
container won't start.

вт, 25 июн. 2024 г. в 08:45, verystrongvery ***@***.***>:
